### PR TITLE
[PHP] Sunrise Router

### LINKED ISSUE #3061: [PHP] Add http-router
Hi @fenric @peter279k,

Would you mind to add (or consent that we add) [http-router](https://github.com/sunrise-php/http-router) in `here` ?

This project compare performance based on 3 routes :

+ `GET` `/`, **SHOULD** return a successful status code and an empty string
+ `GET` `/user/:id`, **SHOULD** return a successful status code and the id
+ `POST` `/user`, **SHOULD** return a successful status code and an empty string

Regards,

### PR DESCRIPTION
**Pull request by fenric**
-----
Closes #3061

### CODE REVIEW

File: php/sunrise-router-annotations/composer.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "require": {
+        "sunrise/http-router": "^2.4",
```
Comment (waghanza) [inline php/sunrise-router-annotations/composer.json:3]: could you use  `~2.4.0`, to ensure in **2.4**

File: php/sunrise-router-annotations/composer.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+        "sunrise/http-factory": "^1.0"
```
Comment (waghanza) [inline php/sunrise-router-annotations/composer.json:4]: could you use  `~1.0.0`, to ensure in **1.0**

File: php/sunrise-router-annotations/config.yaml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+framework:
+  github: sunrise-php/http-router
+  version: 2.4.2
```
Comment (waghanza) [inline php/sunrise-router-annotations/config.yaml:3]: could you use `X.Y` -> `2.4`

File: php/sunrise-router/composer.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+    "require": {
+        "sunrise/http-router": "^2.4",
```
Comment (waghanza) [inline php/sunrise-router/composer.json:3]: could you use  `~2.4.0`, to ensure in **2.4**

File: php/sunrise-router/composer.json [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
+        "sunrise/http-factory": "^1.0"
```
Comment (waghanza) [inline php/sunrise-router/composer.json:4]: could you use  `~1.0.0`, to ensure in **1.0**

File: php/sunrise-router/config.yaml [hunk 6]
```diff
@@ -0,0 +1,3 @@
+framework:
+  github: sunrise-php/http-router
+  version: 2.4.2
```
Comment (waghanza) [inline php/sunrise-router/config.yaml:3]: could you use `2.4`

File: php/sunrise-router-annotations/composer.json (hunk 1 above)
Comment (fenric) [reply php/sunrise-router-annotations/composer.json]: mission complete

File: php/sunrise-router-annotations/composer.json (hunk 2 above)
Comment (fenric) [reply php/sunrise-router-annotations/composer.json]: mission complete

File: php/sunrise-router-annotations/config.yaml (hunk 3 above)
Comment (fenric) [reply php/sunrise-router-annotations/config.yaml]: mission complete

File: php/sunrise-router/composer.json (hunk 4 above)
Comment (fenric) [reply php/sunrise-router/composer.json]: mission complete

File: php/sunrise-router/composer.json (hunk 5 above)
Comment (fenric) [reply php/sunrise-router/composer.json]: mission complete

File: php/sunrise-router/config.yaml (hunk 6 above)
Comment (fenric) [reply php/sunrise-router/config.yaml]: mission complete

**Comment by fenric** [comment]
> forget -> could you add to `CI` in `travis.yml` ?

did I do everything right?

**Pull request merged**